services/deployment-agent/requirements/prod.txt

```diff
@@ -0,0 +1,5 @@
+
+# paths relative to location of setup.py
+.
+#./client-sdk
+git+https://github.com/ITISFoundation/osparc-simcore.git#subdirectory=packages/service-library
```

Comment:
interesting ... now we might think of having release versions of service-library?

Reply:
we'll probably have to think of it as it is used everywhere..